packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -417,7 +417,11 @@ const takeScreenshot = (Cypress, state, screenshotConfig, options: TakeScreensho
     ? subject
     : $dom.wrap(state('document').documentElement)
 
-  return before($el)
+  // get the current body of the AUT to accurately calculate screenshot blackouts
+  // as well as properly enable/disable CSS animations while screenshotting is happening
+  const $body = Cypress.$('body')
```

Comment:
is there possibly a cleaner way to reference the body here?

Reply:
I think this is fine - since the `Cypress` instance is injected to the AUT, and you can only have one instance of `HTMLBodyElement` (or the browser will raise an error) this seems safe to me.